src/components/button/button.yaml

```diff
@@ -31,18 +31,21 @@ examples:
     text: Explicit button disabled
     disabled: true
 - name: button-active-state
+  description: Simulate triggering the :active CSS pseudo-class, not avaliable in the production build.
```

Comment:
ava**il**able

Reply:
when he can only spell because of spellchecker 🙃